### ISSUE TITLE
Add `no-duplicate-schema-methods` rule

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod#198]
### 🧩 Description

There methods in schema that should not called multiple times in a single schema declaration:
- `min`
- `max`
- `trim`
- ...

### ⚙️ Options

None

### 🧾 Example Configuration

No options

### ✅ Valid Code Examples

```typescript
const aSchema = z.string().trim().min(1);
```

### ❌ Invalid Code Examples

```typescript
const aSchema = z.string().trim().min(1).max(5).trim();
```
